spec/lib/task-spec.js

```diff
@@ -242,6 +242,39 @@ describe("Task", function () {
             expect(task.options.nested3).to.equal(definition.options.sourceValue);
         });
 
+        it("should render iteration templates", function() {
+            definition.options = {
+                testList: [
+                    {
+                        name: 'item 1'
+                    },
+                    {
+                        name: 'item 2'
+                    },
+                    {
+                        name: 'item 3'
+                    }
+                ],
+                testVal: '{{#options.testList}}{{ name }}.{{/options.testList}}'
+            };
+            var task = Task.create(definition, {}, {});
+            var tempList = _.transform(definition.options.testList, function (result, n) {
```

Comment:
Yeah we make it available globally in the test helper (https://github.com/RackHD/on-core/blob/master/spec/helper.js#L15), but the .jshintrc doesn't mark global._ as a lint exception. That said, I'd like to try moving away from using too many test globals, so if you don't mind switching to `var _ = helper.injector.get('_')` that would be nice.